src/CarsDataTable.js

```diff
@@ -1,12 +1,23 @@
 import React from 'react';
+import Car from "./Car";
```

Comment:
This bootstrapped project doesn't have any style guide - something we'll need to look into in the future. However, we should always aim and styling consistency - you should use single quotes if that's the coding style of the project you're editing.